packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -0,0 +1,160 @@
+
+### What is Application Insights?
+
+At a high level, App Insights is an Azure cloud service for aggregating, visualizing, analyzing, and alerting on metrics related to a given “service” or “application”.
+You create an App Insights Instance and then configure your applications to send data to your instance using either Azure provided SDK’s or REST APIs.
+This could be general machine related health being automatically reported to the instance when you install a logging program on your service’s machines or custom metrics that you manually configure your applications to send. Keep in mind this logger is intended for use with browser based web applications, not pure nodeJS.
+In our case, we are sending custom metrics. [Learn more about Azure App Insights with their docs](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview?tabs=net)
+
+### Areas currently supported for Fluid telemetry
+
+1. Containers
+
+# Getting Started
+
+Let's walk through some simple examples for getting started with Fluid telemetry for containers using the @fluidframework/fluid-telemetry package, we'll have to write some code.
+
+## Example 1: Logging container telemetry to the console.
+
+In this example, you'll walk through the basic setup process to start getting container telemetry to be produced and logging it to the console.
+
+### Step 1: First, we'll have to create our own telemetry consumer which extends the ITelemetryConsumer interface. Let's look at an example that will simply console.log the telemetry.
+
+```ts
+import { ITelemetryConsumer } from "@fluidframework/fluid-telemetry";
+
+class MySimpleTelemetryConsumer implements ITelemetryConsumer {
+	constructor(private readonly appInsightsClient: ApplicationInsights) {}
+
+	consume(event: IExternalTelemetry) {
+		console.log(event);
+	}
+}
+```
+
+### Step 2: Now, let's start the telemetry production and hook in our telemetry consumer from step 1. We will be initializing our telemetry collection where we initialize our containers:
+
+```ts
+import { IFluidContainer } from "@fluidframework/fluid-static";
+import { ITelemetryConsumer , TelemetryConfig, startTelemetry, IFluidTelemetry } from "@fluidframework/external-telemetry"
+
+// 1: This is supposed to be your code for loading a Fluid Container
+let myAppContainer: IFluidContainer;
+let myAppContainerId: string;
+if (containerExists) {
+    myAppContainerId = {...your code to get the id of the existing container}
+    myAppContainer = {...your code to load a Fluid Container from myAppContainerId}
+} else {
+    myAppContainer = {...your code to create a new Fluid Container}
+    myAppContainerId = await myAppContainer.attach();
+}
+
+// 2: This is our implementation of ITelemetryConsumer
+class MySimpleTelemetryConsumer implements ITelemetryConsumer {
+    constructor(private readonly appInsightsClient: ApplicationInsights) {}
+
```

Comment:
The console.log example doesn't need the app insights client.
```suggestion
```

Reply:
Updated.